be/src/olap/schema_change.h

```diff
@@ -273,8 +279,8 @@ class SchemaChangeHandler {
 
     OLAPStatus schema_version_convert(TabletSharedPtr ref_tablet,
                                       TabletSharedPtr new_tablet,
-                                      std::vector<SegmentGroup*>* ref_segment_groups,
-                                      std::vector<SegmentGroup*>* new_segment_groups);
+                                      std::vector<RowsetSharedPtr>* ref_rowsets,
```

Comment:
new_rowsets may be corresponds to old_rowsets. So old_rowsets may be a better name.